frontend/src/fixtures/currentUserFixtures.js

```diff
@@ -1,136 +1,138 @@
 const apiCurrentUserFixtures = {
-    adminUser: {
-        "user": {
-            "id": 1,
-            "email": "phtcon@ucsb.edu",
-            "googleSub": "115856948234298493496",
-            "pictureUrl": "https://lh3.googleusercontent.com/-bQynVrzVIrU/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucmkGuVsELD1ZeV5iDUAUfe6_K-p8w/s96-c/photo.jpg",
-            "fullName": "Phill Conrad",
-            "givenName": "Phill",
-            "familyName": "Conrad",
-            "emailVerified": true,
-            "locale": "en",
-            "hostedDomain": "ucsb.edu",
-            "admin": true
```

Comment:
Just as a side note, if possible, try to avoid these kinds of  large modifications (makes it harder for the reviewer to see what actually changed)

Reply:
Sorry about that, my autoformatter changes a lot of the spacing.

Reply:
Don't worry about it, just leaving it as a side note